src/containers/Transactions/BreakDownTab/index.tsx

```diff
@@ -0,0 +1,312 @@
+import { FC, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import parsePayment from 'xrpl-tx-path-parser'
+import { Account } from '../../shared/components/Account'
+import { Amount } from '../../shared/components/Amount'
+import { formatAmount } from '../../../rippled/lib/txSummary/formatAmount'
+import Currency from '../../shared/components/Currency'
+
+import './breakDownTab.scss'
+
+// import {
+//   createSourceAmount,
+//   createPaymentDefaultPaths,
+//   amountToBalance,
+// } from 'xrpl-tx-path-parser'
+
+export const BreakDownTab: FC<{ data: any }> = ({ data }) => {
```

Comment:
Any objection to adding this breakdown to the existing "Detailed" tab?

Reply:
I agree